communities/microgalaxy/lab/sections/4_tools.yml

```diff
@@ -7,7 +7,7 @@ tabs:
   content:
   - title_md: Request a new tool or installation of tool
     description_md: "If one of the tools listed here can not be open, it means it
-      is not installed on this server. Please contact the <a href=\"{{ support_url
+      is not installed on this server. Please contact the <a href=\"{{ support
```

Comment:
Here, it should be a direct link to the Help for each server, not the general IMHO.
`https://galaxyproject.org/support/` should be linked only in `General Galaxy Support`

Reply:
then lets do it like that, support for all except france will direct to: https://help.galaxyproject.org and for france it will direct to https://community.france-bioinformatique.fr/c/galaxy/8